README.md

```diff
@@ -1,5 +1,5 @@
 # Simplenote for Android
-[![Version](https://img.shields.io/badge/version-2.18-blue.svg)](https://github.com/Automattic/simplenote-android/releases/tag/2.18) [![CircleCI](https://img.shields.io/circleci/build/gh/Automattic/simplenote-android.svg?label=circleci)](https://circleci.com/gh/Automattic/simplenote-android) [![Travis CI](https://img.shields.io/travis/Automattic/simplenote-android/develop.svg?label=travisci)](https://travis-ci.org/Automattic/simplenote-android)
+[![Version](https://img.shields.io/badge/version-2.19-blue.svg)](https://github.com/Automattic/simplenote-android/releases/tag/2.19) [![CircleCI](https://img.shields.io/circleci/build/gh/Automattic/simplenote-android.svg?label=circleci)](https://circleci.com/gh/Automattic/simplenote-android) [![Travis CI](https://img.shields.io/travis/Automattic/simplenote-android/develop.svg?label=travisci)](https://travis-ci.org/Automattic/simplenote-android)
```

Comment:
Do we still use Travis on this repo? We should probably remove that badge at some point 🤔  (in a separate PR of course, this is not the object of this one)

Reply:
I don't think we do. I'll try to remember to take care of that when I switch it over to Buildkite. Thanks for noting it.

Reply:
My bad, I didn't notice that when removing Travis while back.

Addressed in #1455.